### ISSUE TITLE
NuGet missed dependency to Newtonsoft.Json

### ISSUE THREAD

**Issue by SanyoGorbunov** [docusign/DocuSign-.NET-Client#68]

**Comment by InbarGazit** [comment]
oops, sorry about that @SanyoGorbunov we'll get a new version out and fix this. Let me ask you also about .NET version. We updated when we moved from 1.6.1 to 1.6.2 to 4.5 from 4.0. Do you think we should go back to 4.0?

**Comment by InbarGazit** [comment]
released version 1.7.2 with the dependency to newtonsoft.json is back now
I also decided to change it back to .NET version 4.0 for now. I believe there's a way to have both versions side-by-side, I'll research how to do that later. thanks.

**Comment by SanyoGorbunov** [comment]
Thank you for quick response .) I tried it out for my own test project so didn't have any impartial reasons to vote for 4.0 or 4.5 framework. In production we work with framework 3.5, as far as I understand, we can't use sdk on this platform. Also, noticed, when I nugetted your client at first, used 3.5 framework and this setting was mentioned in packages.config file, but downloaded dll was built for 4.5. So when I tried to build solution, it brought "missing namespace" exception. Good luck !
